shell.nix

```diff
@@ -45,6 +45,7 @@ let
     rm -rf node_modules
     rm -rf typechain
     rm -rf bin
+    rm -rf crytic-export
```

Comment:
does this need to be added to git ignore also?

Reply:
Yup. You're right